index.html

```diff
@@ -0,0 +1,534 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/normalizer.css">
+
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+  integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous"/>
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <nav class="width100 flex">
+      <div class="container-icons flex">
+        <button class="humburger" type="button" name="button"><i class="fas fa-bars"></i></button>
+        <i class="fas fa-search"></i>
+        <h3>Space and cosmos</h3>
+      </div>
+      <a href="#"><img class="nyt-image" src="pictures/NYT.png" alt="NYTicon"></a> 
+      <div class="buttons_top">
+        <button type="button" name="button">SUBSCRIBE NOW</button>
+        <button type="button" name="button">LOG IN</button>
+      </div>
+    </nav>
+  </header>
+  <a href="#">
+    <div class="offer">
+      <div>
+      </div>
+      <p> Access more for free. </p>
+      <span>EXPAND</span>
+    </div>
+  </a>
+  <main>
+    <div class="main_picture_box Center">
+      <h1>Space Ripples Reveal Big Bang’s <br> Smoking Gun</h1>
```

Comment:
- [ ] Please apply the 'italic' rule for the main header like the original page's header.

![image](https://user-images.githubusercontent.com/60184961/90851467-84ffdf80-e37d-11ea-9791-6eb12249e007.png)